app/views/admin/worldwide_offices/index.html.erb

```diff
@@ -11,8 +11,8 @@
       <h2>Offices</h2>
       <div class="tabbable">
         <ul class="nav nav-tabs">
-          <li class="active"><a href="#manage-all" data-toggle="tab">All</a></li>
-          <li><a href="#on-home-page" data-toggle="tab">Order on home page</a></li>
+          <li class="active"><a class="govuk-link" href="#manage-all" data-toggle="tab">All</a></li>
```

Comment:
This looks like an admin template, I didn't think we were applying the layout there, should this be changing?

Reply:
Good spot! Indeed an admin template, will revert the change in this file.